### ISSUE TITLE
Training instability

### ISSUE THREAD

**Issue by Florian-Barthel** [hse1032/GSGAN#3]
Hi,
I am currently playing around with your method and noticed that sometimes the training gets suddenly very unstable (like in the screenshot below). Did you experience the same issue? And do you have an idea how to avoid this? 

(training on 256 resolution with the FFHQ+LPFF dataset)

![image](https://github.com/user-attachments/assets/1ee600ad-1801-4005-976d-24518df0e6c0)

**Comment by hse1032** [comment]
Have you considered increasing the number of gaussians at the coarsest level?

There are no quantitative experiments about it, but I think it will help to enhance the stability.
Besides, you also need to change the hard-coded numbers in lines 603, 607, 618 of "training/point_generator.py" as below.
(I edited them just before)

https://github.com/hse1032/Adversarial-Generation-of-Hierarchical-Gaussians-for-3D-Generative-Model/blob/cc2391f4edfc3a8a08317dc84a33107d42a89535/training/point_generator.py#L603-L618

**Comment by Florian-Barthel** [comment]
Thanks for your help!
I have changed the parameter --gaussian_num_pts to 300 and started a new training. 

And congrats to the NeurIPS acceptance!

**Comment by Florian-Barthel** [comment]
Hi, thanks again for your help! 
I still have some issues reproducing your results. The generator loss always keeps increasing after some time. Did you experience the same in your experiments? And do you use a mirrored version of FFHQ?

**Comment by hse1032** [comment]
Yes. We perfectly follow the data processing of eg3d, so we used mirrored
version of ffhq (i.e. 140k facial images).

Does the generator loss increase rapidly? As I remember, the loss also
increases but it is slowly changed.

If you provide more detailed phenomenon of your failure, I will check it
detail later.

Thanks,


2024년 10월 30일 (수) 오후 7:57, Florian Barthel ***@***.***>님이 작성:

**Comment by Florian-Barthel** [comment]
I have resolved this issue
